### PR TITLE
add tags list page

### PR DESCRIPTION
**Pull request by narthur**

### CODE REVIEW

File: src/components/HOCs/withIntl.spec.tsx [hunk 1]
```diff
@@ -36,7 +37,7 @@ type QueryByText = (
 ) => HTMLElement | null;
 
 const expectNoUnlocalizedText = (queryByText: QueryByText) => {
-	expect(queryByText(/[^z]+/)).not.toBeInTheDocument();
+	expect(queryByText(/[^z0-9]+/)).not.toBeInTheDocument();
```
Comment (mattleff) [inline src/components/HOCs/withIntl.spec.tsx:40]: Would it be better to use a longer, more unique mock formatted message than `z`?

**Review by mattleff — APPROVED** [review]
Looks great! 🍓

File: src/components/HOCs/withIntl.spec.tsx (hunk 1 above)
Comment (narthur) [reply src/components/HOCs/withIntl.spec.tsx:40]: Hmm, in what way?

Comment (mattleff) [reply src/components/HOCs/withIntl.spec.tsx:40]: Right now my brain reads this as "expect text other than `z` or numbers not to be in the document." That's less clear, IMO, than code that would read as "expect text other than `MOCK_LOCALIZED_MESSAGE` or numbers not to be in the document." Currently, it is not at all apparent why the absence of `z` answers/satisfies the function title of "expect no unlocalized text" (unless you already know that `z` is the mock formatted message).

**Pull request merged**